imap_processing/codice/decom.py

```diff
@@ -0,0 +1,22 @@
+""" This is an example of how to use the space_packet_parser module to parse with a XTCE file.
+
+"""
+
+from pathlib import Path
+
+from space_packet_parser import parser, xtcedef
+
+# Define paths
+packet_file = Path("RAW.bin")
+xtce_document = Path("L0/p_cod_aut_test.xml")
+
+packet_definition = xtcedef.XtcePacketDefinition(xtce_document)
+my_parser = parser.PacketParser(packet_definition, 0x460)
```

Comment:
Whenever you have a random number (in this case, `0x460`) that's what we in the biz call a "magic number," because it's just some magical thing you use without explaining. If you find yourself putting numbers in, the best practice is to move the number into a well-named constant variable at the top of your function or file, with a comment explaining what that number is, how it was derived, and what it does. As it is, this is a random number with no context, so I don't know what it does.